src/main/java/org/reficio/p2/P2Mojo.java

```diff
@@ -133,6 +133,16 @@
     private boolean skipInvalidArtifacts;
 
     /**
+     * Skip not existing artifacts.
+     *
+     * <p>
+     * This flag controls if the processing should be continued anyway, if an artifact does not exist. It defaults to
+     * false to keep the old behavior (break on not existing artifacts).
+     */
```

Comment:
Looks good. I'm just thinking that we should maybe also say that it is not only when an artifact does not exist, but in general when a resolution of an artifact fails.